Detectors/Passive/src/Pipe.cxx

```diff
@@ -8,99 +8,2921 @@
+  //
+  // Media
+  const TGeoMedium* kMedAir = GetMedium("PIPE_AIR");
+  const TGeoMedium* kMedAirHigh = GetMedium("PIPE_AIR_HIGH");
+  const TGeoMedium* kMedVac = GetMedium("PIPE_VACUUM");
+  const TGeoMedium* kMedInsu = GetMedium("PIPE_INS_C0");
+  const TGeoMedium* kMedSteel = GetMedium("PIPE_INOX");
+  const TGeoMedium* kMedBe = GetMedium("PIPE_BE");
+  const TGeoMedium* kMedCu = GetMedium("PIPE_CU");
+  const TGeoMedium* kMedAlu2219 = GetMedium("PIPE_AA2219"); // fm
+  const TGeoMedium* kMedRohacell = GetMedium("PIPE_ROHACELL");
+  const TGeoMedium* kMedPolyimide = GetMedium("PIPE_POLYIMIDE");
+  const TGeoMedium* kMedCarbonFiber = GetMedium("PIPE_M55J6K");
+
+  // Top volume
+  TGeoVolume* top = gGeoManager->GetVolume("cave");
+  //
+  //
+  ////////////////////////////////////////////////////////////////////////////////
+  //                                                                            //
+  //                                  The Central Vacuum system                 //
+  //                                                                            //
+  ////////////////////////////////////////////////////////////////////////////////
+  //
+  //
+  //  The ALICE central beam-pipe according to drawing         LHCVC2C_0001
+  //  Drawings of sub-elements:
+  //
+  //  Pos 7 - Minimised Flange:                                LHCVFX_P0025
+  //  Pos 6 - Standard Flange:                                 STDVFUHV0009
+  //  Pos 8 - Bellow:                                          LHCVBX__0001
+  //
+  //  Absolute z-coordinates -82.0 - 400.0 cm
+  //  Total length:                                          482.0 cm
+  //  It consists of 3 main parts:
+  //  CP/2 The flange on the non-absorber side:               36.5 cm
+  //  CP/1 The central Be pipe:                              405.0 cm
+  //  CP/3 The double-bellow and flange on the absorber side: 40.5 cm
+
+  //
+  /*
+  //  Starting position in z
+  const Float_t kCPz0      = -400.0;
+  //  Length of the CP/1 section
+  const Float_t kCP1Length =  405.0;
+  //  Length of the CP/2 section
+  const Float_t kCP2Length =   36.5;
+  //  Length of the CP/3 section
+  const Float_t kCP3Length =   40.5;
+  //  Position of the CP/2 section
+  //    const Float_t kCP2pos    = kCPz0 + kCP2Length / 2.;
+  //  Position of the CP/3 section
+  const Float_t kCP3pos    = kCPz0 + kCP2Length + kCP1Length + kCP3Length/2.;
+  */
+
+  //////////////////// NEW BEAM PIPE GEOMETRY FOR MuonForwardTracker ,
+  // Authors: F. Manso, R. Tieulent
+  // Drawings from C. Gargiulo :
+  // \\cern.ch\dfs\Workspaces\c\cgargiul\EXPERIMENT\ALICE\ALICE_MECHANICS\ALICE_DATA_PACKAGE\IN\DETECTORS\ITS_UPGRADE\1-DESIGN\3D_cad_model\R14_20140311_ALI\
+  //
+  //------------------- Pipe version 4.7 March 2014 -----------------------------
+
+  TGeoVolumeAssembly* beamPipeCsideSection = new TGeoVolumeAssembly("BeamPipeCsideSection");
+
+  Float_t fBeryliumSectionOuterRadius = 1.9;
```

Comment:
the `f` in front probably indicates a float, but since some conventions use this for members of classes this might be misleading. It looks like that several conventions are used in this class, further down one finds similar definitions with `k` prefix, and also some without prefix. Last but not least: should many of those variables be `const` or even `constexpr`?

Reply:
I know. This is indeed highly misleading. For the moment this is a straight copy-paste from AliRoot. Fixing all the variable names will take some time (I need a tool for that).